### ISSUE TITLE
508 compliance issues (Accessibility) APIs page tab order

### ISSUE THREAD

**Issue by christopherwoodland** [Azure/api-management-developer-portal#1383]
## Bug description

Our client has noticed the following accessibility concerns and asked that we submit these concerns to the product team for resolution. These issues violate the 508 compliance, which the customer is bound to.  I am willing to aid in any way to bring these concerns to resolution. Please let me know if additional context or information is required to help move these concerns along.  
I can be reached internally via teams or email cwoodland@microsoft.com
Thank you in advance!

1. On the APIs page, tab focus automatically moves into the Tag Search field when it is opened. However, a user is unable to tab out of the search field popup.  The only way to move tab focus out of the popup is via mouse interaction. This is labeled as high severity.

![image](https://user-images.githubusercontent.com/16299417/125957295-d63f37a6-e876-4d45-94b4-6104b371015a.png)

@mikebudzynski

**Comment by christopherwoodland** [comment]
@jdm4344  Perfect!  I will reference and close these issues.

#1374, #1375, #1378, #1379, #1380.
